### PR TITLE
Package the license file

### PR DESCRIPTION
**Pull request by jakirkham**
Make sure the license file is packaged in `sdist`s and `whl`s.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,2 @@
+include HISTORY.rst
```
Comment (jakirkham) [inline MANIFEST.in]: FYI turns out not having this blocks installation from the `sdist`. However `whl`s do exist, so `pip` typically prefers those.

**Comment by lanpa** [comment]
thanks. I will merge this along with next PyPI release.

**Comment by jakirkham** [comment]
Would be great if this could be merged and tagged even as a patch release. Currently am unable to get a package for 1.1 into conda-forge as there appear to be no git tags for it and the sdist on PyPI is broken without these changes.

**Comment by jakirkham** [comment]
Works great. Thanks. :)

**Pull request merged**